gazebo_plugins/scripts/wheel_slip_drawbar_pull_publisher.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python3
+#
+# Copyright 2022 Open Source Robotics Foundation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Desc: helper script for publishing drawbar pull wrenches
+# Author: Steve Peters
+#
+import argparse
+import math
+import rclpy
+import sys
+from geometry_msgs.msg import Wrench
+from rclpy.node import Node
+
+kDefaultTopic = '/drawbar_pull'
+kDefaultForceIncrement = 10
+kDefaultMaxForce = 70
+kDefaultPublishInterval = 2.0
+
+class WheelSlipDrawbarPullPublisher(Node):
+    def __init__(self, args):
+        super().__init__('wheel_slip_drawbar_pull_publisher')
+        parser = argparse.ArgumentParser(
+            description='Publish drag forces as a wrench representing drawbar pull on the back of a vehicle.')
+        parser.add_argument('-t', '--topic', type=str, default=kDefaultTopic,
+                            help='The topic name for publishing a drawbar pull wrench')
+        parser.add_argument('-f', '--force-increment', type=float, default=kDefaultForceIncrement,
+                            help='The drawbar pull force increment (Newtons)')
+        parser.add_argument('-i', '--interval', type=float, default=kDefaultPublishInterval,
+                            help='The drawar pull publication interval (seconds)')
+        parser.add_argument('-m', '--max-force', type=float, default=kDefaultMaxForce,
+                            help='The maximum drawbar pull force')
+        self.args = parser.parse_args(args[1:])
+
+        self.publisher = self.create_publisher(Wrench, self.args.topic, 1)
+        self.publish_counter = 1
+        self.publish_timer = self.create_timer(self.args.interval, self.updateWrench)
+
+    def updateWrench(self):
+        i = self.publish_counter
+        self.publish_counter += 1
+        amplitude = self.args.force_increment * math.floor(i / 2)
+        sign = (-1) ** (i % 2)
+        if amplitude > self.args.max_force:
+            amplitude = 0
+
+        msg = Wrench()
+        msg.force.x = sign * float(amplitude)
+
+        self.publisher.publish(msg)
+        self.get_logger().info(f"Publishing drawbar pull force of {msg.force.x} N")
```

Comment:
ROS style guide prefers single quotes ([reference](https://docs.ros.org/en/foxy/Contributing/Code-Style-Language-Versions.html#python)).
```suggestion
        self.get_logger().info(f'Publishing drawbar pull force of {msg.force.x} N')
```

Reply:
https://github.com/ros-simulation/gazebo_ros_pkgs/pull/1364/commits/dcca4afd7c4876d96efd84e1c117a926f973b242